_posts/blog/2016-05-05-second-post.adoc

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title: "Second blog post"
+headline: "Checkout this super blog post also..."
```

Comment:
It's just for tests - What I would suggest is to set hidden: true. It won't appear in the website and not in the RSS feed, and it would still be there as an example.